qiime2/metadata/io.py

```diff
@@ -48,7 +48,6 @@ def __init__(self, filepath):
 
         self._filepath = filepath
 
-        # Used by `read()` to store an iterator yielding rows with
```

Comment:
Oops, looks like you deleted part of this pre-existing comment in your changeset.

Reply:
@thermokarst these attempts at using vim commands have me doing that a lot lately :laughing: